eval.go

```diff
@@ -391,22 +553,71 @@ func (st *Runtime) executeList(list *ListNode) {
 					}
 					st.executeLetList(node.Set)
 				} else {
-					st.executeSetList(node.Set)
+					st.executeSetList(node.Set, false)
 				}
 			}
 			if node.Pipe != nil {
 				v, safeWriter := st.evalPipelineExpression(node.Pipe)
 				if !safeWriter && v.IsValid() {
-					if v.Type().Implements(rendererType) {
-						v.Interface().(Renderer).Render(st)
-					} else {
-						_, err := fastprinter.PrintValue(st.escapeeWriter, v)
-						if err != nil {
-							node.error(err)
+					if optionText.isEnabled() == false {
+						if v.Type().Implements(rendererType) {
+							v.Interface().(Renderer).Render(st)
+						} else {
+							_, err := fastprinter.PrintValue(st.escapeeWriter, v)
+							if err != nil {
+								node.error(err)
+							}
 						}
+					} else {
+						tmp := []byte(fmt.Sprintf("%v", v.Interface()))
+						optionText.SetText(tmp)
 					}
 				}
 			}
+		case NodeDefault:
+			node := node.(*DefaultNode)
+
```

Comment:
Can you drop the surrounding empty lines here and also in the following case statements?